### ISSUE TITLE
Bottom compartment of the coffee table is invisible through glass

### ISSUE THREAD

**Issue by heyjudek** [Juuxel/Adorn#124]
### Versions of Minecraft, Fabric and Adorn
- Adorn: 1.12.2
- Minecraft: 1,16,4
- Fabric Loader: 0.10.8
- Fabric API: 0.25.0+build.415-1.16

### Describe the bug
Bottom compartment of the coffee table is invisible through glass. See the picture below - https://i.ibb.co/SQsNv3r/2020-12-22-07-50-26.png

### To Reproduce
1. Craft any coffee table'
2. Look at the bottom compartment through the glass on top
3. The bottom compartment will be invisible
4. Looking at it directly works perfectly fine.

### Expected behavior
The bottom compartment should be visible all the time.

**Comment by Juuxel** [comment]
Hmm, I haven't seen this before. Have you got Optifine installed? Also, does it happen without the custom resource pack?

**Comment by heyjudek** [comment]
> Hmm, I haven't seen this before. Have you got Optifine installed? Also, does it happen without the custom resource pack?

I don't have Optifine. I do have Sodium installed though, could this be the issue? I can see objects through glass in other items with no issues.
The same problem persists without any resource packs too.
https://i.ibb.co/tBRq7Sv/2020-12-23-22-51-47.png
As you see, you cannot see the bottom compartments of the same coffee table if you are lookingat it through its glass, but the bottom compartment of the adjatent coffee table is visible.

**Comment by Juuxel** [comment]
Yeah, it looks like Sodium causes this. I think it's related to jellysquid3/sodium-fabric#38 -- there's not much I can do.

**Comment by Juuxel** [comment]
Reopening this since it's good for actually keeping track of the issue.
